cli/src/main.rs

```diff
@@ -192,10 +192,68 @@ fn run() -> Result<(), CliError> {
                                 .help("Path to private key file"),
                         )
                         .arg(
-                            Arg::with_name("path")
+                            Arg::with_name("node")
+                                .long("node")
                                 .takes_value(true)
                                 .required(true)
-                                .help("Path to a yaml file that defines the circuit proposal"),
+                                .multiple(true)
+                                .help(
+                                    "Node that are part of the circuit. \
+                                  Format: <node_id>::<endpoint>. \
+                                  Endpoint is optional is node alias has been set.",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("service")
+                                .long("service")
+                                .takes_value(true)
+                                .multiple(true)
+                                .min_values(2)
+                                .required(true)
+                                .help(
+                                    "Service ID and allowed node. \
+                                Format <service-id>::<allowed_nodes>",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("service_argument")
+                                .long("service-argument")
+                                .takes_value(true)
+                                .multiple(true)
+                                .help(
+                                    "Special arguments to be passed to the service. \
+                                Format <service_id>::<key>=<value>",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("management_type")
+                                .long("management-type")
+                                .takes_value(true)
+                                .help("Management type for a service"),
+                        )
+                        .arg(
+                            Arg::with_name("authorization_type")
+                                .long("auth-type")
+                                .possible_values(&["trust"])
+                                .default_value("trust")
+                                .takes_value(true)
+                                .help("Authorization type for a service"),
```

Comment:
auth type is for the circuit, not a service

Reply:
Updated